docs/video/ffmpeg/upscaling.md

```diff
@@ -50,12 +50,13 @@ Here are two videos, one was left as is, the other one stretched to 4K, watch it
     
 === "1080p stretched to 4K"
 
-    <iframe width="688" height="387" src="https://www.youtube.com/embed/-FE9s_acdYw?color=white" frameborder=0 allowfullscreen></iframe>
+    <iframe width="688" height="387" src="https://www.youtube.com/embed/-FE9s_acdYw" frameborder=0 allowfullscreen></iframe>
 
     
 ## **Why not just set the project scale to 4K before rendering?** {#why-not-nle}
 
-VEGAS Pro (and probably other NLEs) uses the bicubic scaling filter which makes your video a bit blurry, using FFmpeg lets you make sure you're using the right filter
+VEGAS Pro (and probably any other NLE) uses the bicubic scaling filter which makes your video a bit blurry, using FFmpeg lets you make sure you're using the right filter
 
-> :custom-voukoder: **Why not use Voukoder's zscale?**
-__You ideally **should** use it __over the batchscript you can install in this guide, since it will export your project and upscale it all at once, which should take less time and lose less efficiency, more info @ <#1022956596768288919>
\ No newline at end of file
+## :custom-voukoder: **Why not use Voukoder's zscale?** {#why-not-voukoder}
+
+You ideally **should** use it over the batchscript you can install in this guide, since it will export your project and upscale it all at once, which should take > less time and lose less efficiency, more info on Discord @ <#1022956596768288919>
```

Comment:
i gotta update this discord link <#1022956596768288919> i'll do it myself thanks